UACloudLibraryServer/UA-CloudLibrary.csproj

```diff
@@ -32,52 +32,55 @@
 	</ItemGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Amazon.AspNetCore.DataProtection.SSM" Version="3.0.0" />
-		<PackageReference Include="AWSSDK.Extensions.NETCore.Setup" Version="3.7.2" />
-		<PackageReference Include="AWSSDK.S3" Version="3.7.9.69" />
-		<PackageReference Include="AWSSDK.SecurityToken" Version="3.7.1.215" />
-		<PackageReference Include="Azure.Extensions.AspNetCore.DataProtection.Blobs" Version="1.2.3" />
-		<PackageReference Include="Azure.Storage.Blobs" Version="12.14.0" />
-		<PackageReference Include="Google.Cloud.Storage.V1" Version="4.1.0" />
+		<PackageReference Include="Amazon.AspNetCore.DataProtection.SSM" Version="3.1.0" />
+		<PackageReference Include="AWSSDK.Extensions.NETCore.Setup" Version="3.7.7" />
+		<PackageReference Include="AWSSDK.S3" Version="3.7.104.10" />
+		<PackageReference Include="AWSSDK.SecurityToken" Version="3.7.101.60" />
+		<PackageReference Include="Azure.Extensions.AspNetCore.DataProtection.Blobs" Version="1.3.2" />
+		<PackageReference Include="Azure.Storage.Blobs" Version="12.16.0" />
+		<PackageReference Include="Google.Cloud.Storage.V1" Version="4.5.0" />
 		<PackageReference Include="Google.Cloud.AspNetCore.DataProtection.Storage" Version="1.0.0-alpha03" />
 	</ItemGroup>
 	<ItemGroup>
-		<PackageReference Include="Microsoft.EntityFrameworkCore.Relational" Version="6.0.10" />
-		<PackageReference Include="HotChocolate" Version="13.0.2" />
-		<PackageReference Include="HotChocolate.Data" Version="13.0.2" />
-		<PackageReference Include="HotChocolate.Types.Scalars" Version="13.0.2" />
-		<PackageReference Include="HotChocolate.AspNetCore" Version="13.0.2" />
-		<PackageReference Include="HotChocolate.AspNetCore.Authorization" Version="13.0.2" />
-		<PackageReference Include="CESMII.OpcUa.NodeSetImporter" Version="1.0.11" />
-		<PackageReference Include="CESMII.OpcUa.NodeSetModel.EF" Version="1.0.11" />
-		<PackageReference Include="CESMII.OpcUa.NodeSetModel.Factory.Opc" Version="1.0.11" />
-		<PackageReference Include="CESMII.OpcUa.NodeSetModel" Version="1.0.11" />
+		<PackageReference Include="Microsoft.EntityFrameworkCore.Relational" Version="6.0.16" />
+		<PackageReference Include="HotChocolate" Version="13.0.5" />
+		<PackageReference Include="HotChocolate.Data" Version="13.0.5" />
+		<PackageReference Include="HotChocolate.Types.Scalars" Version="13.0.5" />
+		<PackageReference Include="HotChocolate.AspNetCore" Version="13.0.5" />
+		<PackageReference Include="HotChocolate.AspNetCore.Authorization" Version="13.0.5" />
+		<PackageReference Include="CESMII.OpcUa.NodeSetImporter" Version="1.0.12" />
```

Comment:
This nuget has a breaking change. Need to add 
                orn.Ignore(nr => nr.ReferenceType);
for both OtherReferencedNodes and OtherReferencingNodes, or stay with 1.0.11.

Reply:
I'm going to need your help with that change...

Reply:
Plan is to test with 1.0.14 instead, which no longer has the breaking change.

Reply:
Picked up 1.0.13 (no .14 yet).